blueocean-rest/pom.xml

```diff
@@ -23,6 +23,10 @@
           <groupId>io.jenkins.blueocean.rest.annotation</groupId>
           <artifactId>capability-annotation</artifactId>
       </dependency>
+      <dependency>
```

Comment:
See my other comment, we should avoid depending on specific date time library in models.